moxystudio/react-with-moxy
# Fix CSS modules on built version

With the current config, the css modules weren't being correctly imported into the js used in the server.
To test this:
* log the `styles` variable on any component.
* npm run build:prod and then npm start
* you'll see an empty object being logged on the server and then the correct one being logged in the client
* this causes a FOUC -> try to place a breakpoint on the browser and when it stops there, we can see the unstyled markup (also, a lot of `undefined` on the class attribute of every element)
